next/TilesetWithFullMetadata/README.md

```diff
@@ -0,0 +1,66 @@
+# Tileset with full Metadata
```

Comment:
Given some of my other feedback, we might want to rename this.

Unfortunately we already have `TilesetWithTilesetMetadata` so we'd need another name...